meetings/2021-11-09-Community_Meeting.md

```diff
@@ -0,0 +1,108 @@
+# Gluster Community Meeting - 09/11/2021
+
+
+### Previous Meeting minutes:
+
+- [Meeting Minutes](https://github.com/gluster/community/tree/master/meetings)
+
+### Date/Time: Check the [community calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=MDQ0YmRydTllMXYzdWFoMmpsbjdqNXJlYmNfMjAyMDEwMjdUMDkwMDAwWiBzYWptb2hhbUByZWRoYXQuY29t&tmsrc=sajmoham%40redhat.com&scp=ALL)
+
+### Bridge
+  - 2nd / 4th Tuesday's, 02:30PM IST
+  - Bridge: https://meet.google.com/cpu-eiue-hvk
+
+
+-------
+
+### Attendance
+Name (#gluster-dev alias) - company
+* Saju Mohammed (Red Hat)
+* Pranith Kumar K (PhonePe)
+* Nikhil Ladha - Red Hat
+* Rakshitha Kamath (Red Hat)
+* Ravishankar (Pavilion Data)
+* Sheetal pamecha (Red Hat)
+* Sunil Kumar (Red Hat)
+* Shwetha Acharya (Red Hat)
+
+### Community
+
+* Project metrics:
+
+|    Metrics                |   Value  |
+| ------------------------- | -------- |
+|[Coverity](https://scan.coverity.com/projects/gluster-glusterfs)  | 19 (19)  |
+|[Clang Scan](https://build.gluster.org/job/clang-scan/lastBuild/) |   87 (87)  |
+|[Line coverage](https://build.gluster.org/job/line-coverage/lastCompletedBuild/Line_20Coverage_20Report/)|     70.7% (70.9 %) |
+|[Function coverage](https://build.gluster.org/job/line-coverage/lastCompletedBuild/Line_20Coverage_20Report/)|     84.8 (85.1%) |
+|[Gluster User Queries in last 14 days](https://lists.gluster.org/pipermail/gluster-users/2021-October/thread.html)        |     6      |
+|[Total Github issues](https://github.com/gluster/glusterfs/issues)       |  266     |
+
+
+* Any release updates?
+      
+    * Gluster 10 - Tcmalloc has build issues with other architectures
+        * We will be enabling tcmalloc with x86_64 architecture only. Release will be happening this week.
+
+* Blocker issues across the project?
+    * https://github.com/gluster/glusterfs/pull/2931
+
+* User stories / Notable thread from the mailing list
+    * 
```

Comment:
https://aws.amazon.com/blogs/hpc/scaling-a-read-intensive-low-latency-file-system-to-10m-iops is worth mentioning?

Reply:
Already put it under "Recent Blog posts" down the page.